### PR TITLE
intl-tel-input imported

### PR DESCRIPTION
**Pull request by PoisoN-ck**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import $ from 'jquery';
 import 'slick-carousel';
+import 'intl-tel-input/index.js';
```
Comment (AlexeyPogorelov) [inline src/js/main.js:4]: Добавь остальной код

**Review by AlexeyPogorelov — CHANGES_REQUESTED** [review]

File: src/js/main.js (hunk 1 above)
Comment (PoisoN-ck) [reply src/js/main.js:4]: Добавил

**Review by AlexeyPogorelov — APPROVED** [review]

File: src/form.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/build.css">
+</head>
+<body>
+    <header></header>
+    <section>
+        <h1 style="text-align: center">Order now</h1>           
+        <form class="orderNow" name="Order" action="" method="post" autocomplete="off">
+            <label for="firstName"></label>
```
Comment (AlexeyPogorelov) [inline src/form.html]: create group for each label / input

File: src/form.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            <input id="firstName" type="text" size=65 name="First Name" placeholder="First Name" required> <br>        
+            
+            <label for="lastName"></label>
+            <input id="lastName" type="text" size=65 name="Last Name" placeholder="Last Name" required> <br>                
+            
+            <label for="userEmail"></label>
+            <input id="userEmail" type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" size=65 name="E-mail" placeholder="ihor.test@mail.com" autocomplete="on" required> <br>
+
+            <label for="telNum"></label>
+            <input id="telNum" type="tel" size=55 name="Phone" required>
```
Comment (AlexeyPogorelov) [inline src/form.html]: use attributes for autocomplete

File: src/js/form.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
+import intlTelInput from 'intl-tel-input/index.js';
+import 'intl-tel-input/build/js/utils';
+import 'flatpickr';
+import rangePlugin from '../../node_modules/flatpickr/dist/plugins/rangePlugin';
```
Comment (AlexeyPogorelov) [inline src/js/form.js]: Have you tried to get this plugin by absolute path? `flatpickr/dist/plugins/rangePlugin`

File: src/js/form.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+let countries_served = ["az", "ua", "ge"];
+
+function makeTitle(str) {
```
Comment (AlexeyPogorelov) [inline src/js/form.js]: could you get same effect using CSS?

File: src/js/form.js [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+        element.value = input;
+        return element.value;
+    })
+}
+
+function restrictKeyboardInput(element) {
+    element.addEventListener("keydown", function(e) {
+        if (e.keyCode === 13) {
+            document.getElementById("peopleCount").focus();
+        } else if (e.keyCode === 9 || e.keyCode === 16) {
```
Comment (AlexeyPogorelov) [inline src/js/form.js:24]: combine both cases in one if

File: src/js/form.js [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    "locale": {
+        "firstDayOfWeek": 1
+    },
+    "plugins": [new rangePlugin({input: checkOutDate})],
+    allowInput: true
+});
+
+saveDates(checkInDate, checkInDate, checkOutDate);
+saveDates(checkOutDate, checkInDate, checkOutDate);
+
```
Comment (AlexeyPogorelov) [inline src/js/form.js]: empty lines

File: src/sass/form.scss [hunk 8]
```diff
@@ -0,0 +1,29 @@
+.iti-flag {
+    background-image: url("/img/flags.png");
```
Comment (AlexeyPogorelov) [inline src/sass/form.scss:2]: use relative path `./img/...`

File: src/sass/form.scss [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```
Comment (AlexeyPogorelov) [inline src/sass/form.scss:5]: try to not use browser prefix. Look at SASS output then

File: src/sass/form.scss [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+.datesWrapper {
+    display: flex;
+    justify-content: start;
+}
+
+.checkDates {
+    margin-right: 50px;
+    width: 30%;
+}
```
Comment (AlexeyPogorelov) [inline src/sass/form.scss]: leave empty line at the end of the file

File: src/form.html [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+            </select>
+            
+            <label for="comments"></label>
+            <textarea id="comments" placeholder="Your message" cols="55" name="Comments"></textarea> 
+            
+            <div>
```
Comment (AlexeyPogorelov) [inline src/form.html:43]: ???

File: src/form.html [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+                <input id="submit" type="submit" placeholder="send" name="Submit">
+                <input id="reset" type="reset" placeholder="Reset" name="Reset">    
+            </div>
+            
+        </form> 
+    </section>    
+    <footer></footer>
```
Comment (AlexeyPogorelov) [inline src/form.html]: ???

File: src/js/form.js [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+function saveDates(element, checkIn, checkOut) {
+    element.addEventListener("change", function() {
+        checkInDateValue = checkIn.value;
+        checkOutDateValue = checkOut.value;
+        return checkInDateValue, checkOutDateValue;
+    })
+}
+
+//First name validation
+let firstName = document.getElementById("firstName");
```
Comment (AlexeyPogorelov) [inline src/js/form.js]: use querySelector. It looks more general and clean in most cases

**Review by AlexeyPogorelov — CHANGES_REQUESTED** [review]

File: README.md [hunk 14]
```diff
@@ -1,2 +1,2 @@
 # Alison travel
-Layout for website.
+Country form validation for the website. Reach on localhost:post/form.html
```
Comment (AlexeyPogorelov) [inline README.md]: This is description for whole repository. Not for your branch.

**Review by AlexeyPogorelov — CHANGES_REQUESTED** [review]

File: src/js/form.js (hunk 6 above)
Comment (PoisoN-ck) [reply src/js/form.js:24]: behaviour expected has to be different in those 2 cases, enter -> focus on next field, tab or tab-shift -> move between inputs
```suggestion
        } else if (e.keyCode === 9 || e.keyCode === 16) {
```

File: src/sass/form.scss (hunk 8 above)
Comment (PoisoN-ck) [reply src/sass/form.scss:2]: with relative path doesn't work, used this way to reflect url path

File: src/sass/form.scss (hunk 9 above)
Comment (PoisoN-ck) [reply src/sass/form.scss:5]: Noted

**Comment by PoisoN-ck** [comment]
Please check fixes and advise

File: src/sass/build.sass [hunk 15]
```diff
@@ -37,6 +37,9 @@
 @import "../../node_modules/bootstrap/scss/print"
 @import "../../node_modules/slick-carousel/slick/slick.scss"
 @import "../../node_modules/slick-carousel/slick/slick-theme.scss"
+@import "../../node_modules/intl-tel-input/src/css/intlTelInput.scss"
```
Comment (AlexeyPogorelov) [inline src/sass/build.sass]: you can remove .scss

File: src/js/form.js [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+    minDate: "today",
+    "locale": {
+        "firstDayOfWeek": 1
+    },
+    "plugins": [new rangePlugin({input: checkOutDate})],
+    allowInput: true
+});
+
+saveDates(checkInDate, checkInDate, checkOutDate);
+saveDates(checkOutDate, checkInDate, checkOutDate);
```
Comment (AlexeyPogorelov) [inline src/js/form.js]: empty line

File: src/sass/form.scss [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+    background-image: url("/img/flags.png");
+}
+
+@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
+        .iti-flag {
+            background-image: url("/img/flags.png");
+    }
+}
+
+#firstName, #lastName {
```
Comment (AlexeyPogorelov) [inline src/sass/form.scss]: do not use ID for styles!

File: src/form.html [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+    <title>Alison. Form validation</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/build.css">
+</head>
+<body>
+    <section>
+        <h1 style="text-align: center">Order now</h1>           
+        <form class="orderNow" name="Order" action="" method="post" autocomplete="off">
```
Comment (AlexeyPogorelov) [inline src/form.html:14]: name???

File: src/form.html [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+    <meta charset="UTF-8">
+    <title>Alison. Form validation</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/build.css">
+</head>
+<body>
+    <section>
+        <h1 style="text-align: center">Order now</h1>
```
Comment (AlexeyPogorelov) [inline src/form.html:13]: do not use inline styles

**Review by AlexeyPogorelov — CHANGES_REQUESTED** [review]

File: src/form.html [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/build.css">
+</head>
+<body>
+    <section>
+        <h1 style="text-align: center">Order now</h1>           
+        <form class="orderNow" name="Order" action="" method="post" autocomplete="off">
+            <div>
+                <label for="firstName"></label>
```
Comment (Alex-Kostenko) [inline src/form.html:16]: пустые лейблы не используются

File: src/form.html [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+                <input id="firstName" type="text" size=65 name="First Name" placeholder="First Name" required>
+            </div>
+
+            <div>
```
Comment (Alex-Kostenko) [inline src/form.html:20]: empty div

**Review by Alex-Kostenko — CHANGES_REQUESTED** [review]

**Comment by Alex-Kostenko** [comment]
many extra spaces at the end of lines

File: src/form.html (hunk 21 above)
Comment (PoisoN-ck) [reply src/form.html:20]: used for readability, fixed

**Comment by PoisoN-ck** [comment]
Added suggestions to a code

**Review by Alex-Kostenko — APPROVED** [review]

**Pull request merged**